StarryExpanse/EverPresent/StrangerController.h

```diff
@@ -57,6 +57,14 @@ class STARRYEXPANSE_API AStrangerController : public APlayerController {
   UPROPERTY()
   float VerticalMousePosition = 0.5;
 
+  //how close (%) to the edge of the left/right of the screen you can get (in cursor mode) before the camera begins to pan
```

Comment:
Since these are fractions, let's not call them %. Percent would imply we are dividing by 100 before multiplication when in fact we are just multiplying.

Reply:
good catch.  updated comments.